Applications/DataExplorer/DataView/DiagramView/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ set(SOURCES
 
 set(HEADERS
     DiagramList.h
+    getDateTime.h
```

Comment:
I'd use capital letter in the beginning for consistency: `GetDateTime.h`
And you forgot to upload the actual file, I guess...